Orange/widgets/data/owpivot.py

```diff
@@ -821,6 +829,11 @@ def commit(self):
         self.Outputs.pivot_table.send(self.pivot.pivot_table)
         self.Outputs.filtered_data.send(self.get_filtered_data())
 
+        output = self.get_filtered_data()
```

Comment:
Since you are saving the filtered data into a temporary variable, you can do it before `self.Outputs.filtered_data.send(self.get_filtered_data())` and use it there as well instead of crating the data twice.